File: src/easy/numberfun.py
for _ in range(int(input())):
    a, b, c = [int(d) for d in input().split()]
    if a + b == c or a * b == c or a - b == c or b - a == c or a / b == c or b / a == c:
        print("Possible")
    else:
        print("Impossible")

File: src/numberfun.lua
for _ = 1, io.read("*n") do
    a, b, c = io.read("*n", "*n", "*n")
    if a + b == c or a * b == c or a - b == c or b - a == c or a == b * c or b == a * c then
        print("Possible")
    else
        print("Impossible")
    end
end
